dns.go

```diff
@@ -126,9 +130,17 @@ func probeDNS(target string, w http.ResponseWriter, module Module, registry *pro
 		fallbackProtocol = "ip6"
 	}
 
+	targetAddress, targetPort, _ = net.SplitHostPort(target)
+	if targetPort == "" {
+		// missingPort could be fixed by fallbacking to the default port.
```

Comment:
I'm not sure about this line, as [SplitHostPort](https://golang.org/src/net/ipsock.go?s=3572:3638#L104) func has `missingPort ` and `tooManyColons ` error, will the wrong ipv6 address trouble our probe?

I'm afraid about whether there would be potential panic here...